.github/workflows/ci.yml

```diff
@@ -122,28 +122,28 @@ jobs:
             ${{ runner.os }}-yarn-
 
       - name: Install dependencies
-        run: yarn install --frozen-lockfile --ignore-engines --prefer-offline
```

Comment:
We upgraded from yarn 1 to yarn 3 and `--ignore-engines --prefer-offline` have been removed